### PR TITLE
fix(core): `BreakpointService` replay value for new subscribers

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -54,7 +55,7 @@
     },
     "lint-staged": {
         "*.{js,ts,html,md,less,json}": [
-            "npm run lint -- --fix",
+            "npm run lint-precommit -- --fix",
```
Comment (splincode) [inline package.json:58]: just

```
eslint --fix
```

File: package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
         "*** Workflow ***": "",
         "stylelint": "nx run-many --target stylelint --all --output-style=stream --parallel=1",
         "lint": "nx run-many --target lint --all --output-style=stream --parallel=1",
+        "lint-precommit": "eslint --cache --cache-location node_modules/.cache/eslint",
```
Comment (splincode) [inline package.json:39]: no needed

**Review by waterplea — APPROVED** [review]

**Comment by zargham-leanix** [comment]
Hi, I am wondering what difference does `share` and `shareReplay` make here ?

**Comment by waterplea** [comment]
`share` shares the subscriptions, `shareReplay` also shares the last emitted value.

> Hi, I am wondering what difference does `share` and `shareReplay` make here ?

**Pull request merged**